contrib/depdot.sh

```diff
@@ -1,12 +1,11 @@
 #!/bin/bash
 
+set -e
 # Script to output the dependency graph of Gluon's packages
 # Limitations:
 #  * Works only if directory names and package names are the same (true for all Gluon packages)
 #  * Doesn't show dependencies through virtual packages correctly
 
-
-
 shopt -s nullglob
```

Comment:
As set and shopt are closely related, please move the `set -e` down below the comment.